book/types_of_data.md

```diff
@@ -245,10 +301,75 @@ You can also create a table as a list of records:
 ╰───┴──────┴──────╯
 ```
 
-### Column paths
+:::tip
+Internally, tables are simply **lists of records**. This means that any command which extracts or isolates a specific row of a table will produce a record. For example, `get 0`, when used on a list, extracts the first value. But when used on a table (a list of records), it extracts a record:
+```sh
+> [{x:12, y:5}, {x:3, y:6}] | get 0
+╭───┬────╮
+│ x │ 12 │
+│ y │ 5  │
+╰───┴────╯
+```
+This is true regardless of which table syntax you use:
+```sh
+[[x,y];[12,5],[3,6]] | get 0
+╭───┬────╮
+│ x │ 12 │
+│ y │ 5  │
+╰───┴────╯
+```
+:::
+
+
+### Paths
```

Comment:
I believe these are called `Cell Paths` vs just Paths. They used to be called `Column Paths` as well.